EORTC/readme.md

```diff
@@ -0,0 +1,41 @@
+Update of EORTC QLQ Vocabulary
+
+Prerequisites:
+- Vocabulary Server infrastructure
+- Schema DevV5 with copies of tables concept, concept_relationship and concept_synonym from ProdV5, fully indexed.
+- Working directory EORTC
+
+1. Run the vocabulary_download.get_eortc (automated scraping was set-up with twice a year schedule or scraping can be performed on-demand basis)
+   1. The Item Library Web-Site (itemlibrary.eortc.org) version dated Spring-2024 was used for initial scraper set-up;
+   2. The Credentials are linked to OHDSI Vocabulary Account (permission from EORTC team was received via e-mail)
```

Comment:
Please rewrite it to something like
'caution: the content cannot be automatically processed without prior agreement from the EORTC team'

Reply:
Done